bin/uglifyjs

```diff
@@ -44,6 +44,7 @@ program.option("-o, --output <file>", "Output file (default STDOUT).");
 program.option("--comments [filter]", "Preserve copyright comments in the output.");
 program.option("--config-file <file>", "Read minify() options from JSON file.");
 program.option("-d, --define <expr>[=value]", "Global definitions.", parse_js("define"));
+program.option("--ecma <version>", "Specifiy ECMAScript release number: 5, 6, 7 or 8. Default is ES8.");
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2157/commits/33de720c40acced40fc95cdd7a3b004b6c1ac332